concent_api/common/tests/test_decorators.py

```diff
@@ -278,3 +281,20 @@ def task():
 
         mock_format_exc.assert_called_once()
         mock_error.assert_called_once()
+
+
+@override_settings(
+    DETECT_NESTED_TRANSACTIONS=True,
+)
+class NonNestingAtomicDecoratorTestCase(TransactionTestCase):
```

Comment:
could you please add more tests, i.e for cases when `DETECT_NESTED_TRANSACTIONS=False`.

Reply:
:+1: done